### DISCUSSION TITLE
Game title. Is it possible to change it?

### DISCUSSION THREAD

**Discussion by jk7gr** [giongto35/cloud-game#472] [Q&A] [answered]
Considering is not possible to rename fbneo roms is it somehow possible to make the title of the game that is visible different than the actual zip filename?

**Comment by jk7gr** [comment]
Maybe having the option of a txt file where rom filename will be associated with game names. 

So, to create a textfile `alias.txt` in the folder with the roms. In that file to write `'the zip name'='name to be displayed'`, like the example below:

    avspu=Alien vs Predator
    sfau=Street Fighter Alpha
    ssf2tu=Super Street Fighter II Turbo
    ...

**Reply by sergystepanov** [reply, marked as answer]
OK. I'm going to try that right now.

**Reply by sergystepanov** [reply]
Ok, @jk7gr, 
could you try feature/game-alias branch? Put your aliases into the `alias.txt` in the games folder.

![image](https://github.com/user-attachments/assets/eab81958-246a-4b2c-82f5-607e9c0553c1)

**Reply by jk7gr** [reply]
can I use the branch tag in docker? Apologies. I am not that much familiar. I am using the docker image. Maybe you can use the dev docker image tag?

**Reply by sergystepanov** [reply]
I merged it into the master branch, so everything is now in the master Docker image or the master branch.

**Reply by jk7gr** [reply]
Good. I am testing and coming back

**Reply by sergystepanov** [reply]
Don't forget to clear site (browser) cache. In Chrome that's shift+f5

**Reply by jk7gr** [reply]
It seems is not working. Or I am doing something wrong. Maybe is important to mention I am organizing the roms per folder I have placed the alias.txt in both inside the folder but also in the root.

**Reply by jk7gr** [reply]
Sample of my file:

aburner=After Burner
aerofgtb=Aero Fighters
fatfury1=Fatal Fury - King of Fighters - Garou Densetsu - Shukumei no Tatakai
fatfury2=Fatal Fury 2 - Garou Densetsu 2 - Arata-naru Tatakai
fatfury3=Fatal Fury 3 - Road to the Final Victory - Garou Densetsu 3 - Haruka-naru Tatakai
kinst=Killer Instinct
kinst2=Killer Instinct 2
mk2r91=Mortal Kombat II
mk3r20=Mortal Kombat 3
nbajamex=NBA Jam Extreme
pbobblen=Puzzle Bobble - Bust-A-Move
pgoal=Pleasure Goal - Futsal - 5 on 5 Mini Soccer
pspikes2=Power Spikes II
ridhero=Riding Hero
rungun=Run and Gun
rungun2=Run and Gun 2
sf2eb=Street Fighter II: The World Warrior
soccerss=Soccer Superstars
sonicwi=Sonic Wings
sonicwi2=Aero Fighters 2 - Sonic Wings 2
sonicwi3=Aero Fighters 3 - Sonic Wings 3
spool3=Super Pool III
ssideki=Super Sidekicks - Tokuten Ou
ssideki2=Super Sidekicks 2 - The World Championship - Tokuten Ou 2 - Real Fight Football
ssideki3=Super Sidekicks 3 - The Next Glory - Tokuten Ou 3 - Eikou e no Chousen
svolly91=Super Volley '91
tehkanwc=Tehkan World Cup
tws96=Tecmo World Soccer '96
vsnetscr=Versus Net Soccer
mwalkbl=Michael Jackson's Moonwalker

**Reply by sergystepanov** [reply]
**_Did you clear the browser cache?_** It should display [bdf3598](https://github.com/giongto35/cloud-game/commit/bdf3598367607f26edd0444c71ffa99f4c58d2c3) revision in the bottom right corner (if you use master Docker image).

It's enough just to place it in the root of your games dir.
In the file each line should be: exact name of the game as displayed in the list on the main page=whatever you want it to be

You can set the environment variable CLOUD_GAME_COORDINATOR_DEBUG=true or set `coordinator.debug` to true in your config.yaml, then it will print games and their aliases into console (docker logs).

alias.txt
```txt
ddsom=Dungeon & Dragons: Something something
```
![image](https://github.com/user-attachments/assets/792032cc-0bed-4d40-bf83-506a65f3108b)

**Reply by jk7gr** [reply]
Ok. Initial testing is good. Let me put the files in correct structure and extend testing but we have a good start ![Screenshot_20240831_235312_Chrome.jpg](https://github.com/user-attachments/assets/7b7fc773-6822-40c9-a406-c7366d1ebfcd)

**Reply by jk7gr** [reply]
Yes!! It works mate!! Wow, amazing. Well done man. I was having this request almost since the beginning.  Thank you.

**Reply by sergystepanov** [reply]
Haha, you are welcome.
